src/components/RegistrationList.tsx

```diff
@@ -106,13 +102,16 @@ export const RegistrationListItemContent = ({
   return (
     <Box sx={{ display: 'flex', width: '100%', gap: '1rem' }}>
       <ListItemText disableTypography data-testid={dataTestId.startPage.searchResultItem}>
-        <Box sx={{ display: 'flex', alignItems: 'center', gap: { xs: '1rem', sm: '2rem' } }}>
-          {heading && (
-            <Typography variant="overline" sx={{ color: 'primary.main' }}>
-              {heading}
+        <Box sx={{ display: 'flex', alignItems: 'center', gap: { xs: '1rem', sm: '2rem' }, marginBottom: '0.5rem' }}>
+          <Box sx={{ display: 'flex', gap: '0.4rem' }}>
+            <RegistrationIcon />
+            <Typography sx={{ color: 'primary.main' }}>
+              {registrationType ? t(`registration.publication_types.${registrationType}`) : ''}
             </Typography>
```

Comment:
kanskje greit med en sjekk utafor, så vi slipper potensielt tomme `<p></p>`?

```suggestion
            {registrationType && (
              <Typography sx={{ color: 'primary.main' }}>
                {t(`registration.publication_types.${registrationType}`)}
              </Typography>
            )}
```